app/celery/letters_pdf_tasks.py

```diff
@@ -310,6 +310,7 @@ def _upload_pdf_to_test_or_live_pdf_bucket(pdf_data, filename, is_test_letter):
 
 
 def _sanitise_precompiled_pdf(self, notification, precompiled_pdf):
+    response = None
```

Comment:
This leads to an `AttributeError` on line 330. You need it to be like: `if response is not None and if 'message'....`. Additionally, there is no guarantee "message" is in the json, so it should be a safe lookup. We don't want unsafe lookups in our exception handling.